process/smartContract/process.go

```diff
@@ -172,12 +173,12 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 	sw.Stop("execute")
 	duration := sw.GetMeasurement("execute")
 
-	logFunc := log.Trace
-	if duration > executeDurationWarnThreshold {
-		logFunc = log.Debug
+	if duration > executeDurationAlarmThreshold {
```

Comment:
Not for now, but I think that this:
```go
sw := core.NewStopWatch()
sw.Start("execute")
returnCode, err := sc.doExecuteSmartContractTransaction(tx, acntSnd, acntDst)
sw.Stop("execute")
duration := sw.GetMeasurement("execute")

if duration > executeDurationAlarmThreshold {
	log.Debug(fmt.Sprintf("scProcessor.ExecuteSmartContractTransaction(): execution took > %s", duration), "sc", tx.GetRcvAddr(), "duration", duration, "returnCode", returnCode, "err", err, "data", string(tx.GetData()))
} else {
	log.Trace("scProcessor.ExecuteSmartContractTransaction()", "sc", tx.GetRcvAddr(), "duration", duration, "returnCode", returnCode, "err", err, "data", string(tx.GetData()))
}
```

Could become something like this?

```go
func TestStopwatch(t *testing.T) {
        x := 100
	measure(func() {
		fmt.Println(x)
	})
}

type measuredFuc func()
func measure(myFunc measuredFuc) {
	fmt.Println("initStopwatch")
	myFunc()
	fmt.Println("endStopwatch")
}
```

Reply:
Added a Jira task here: https://elrondnetwork.atlassian.net/browse/EN-7026